### PR TITLE
Handle errors deposit withdrawal

### PR DESCRIPTION
**Pull request by gndelia**
Related to #529 

This PR adds error handling and re-tries for Ethereum Deposits and Withdrawals. After this PR, the Sepolia flow should be completed. It is pending to implement it for bitcoin

### Some screenshots

State updated in the transaction history

image

Failed operations are listed if users want to retry them, at any step

image

image

image

### CODE REVIEW

**Review by gabmontes — APPROVED** [review]
uACK

File: webapp/app/[locale]/tunnel/_components/reviewOperation/step.tsx [hunk 1]
```diff
@@ -60,16 +56,23 @@ const Completed = function ({
...
+            <SeeOnExplorer chainId={explorerChainId} txHash={txHash} />
+          </>
+        }
+        top={{
+          bgColor: 'bg-neutral-50',
+          children: (
+            <>
+              <span className="mr-auto text-neutral-600">{description}</span>
+            </>
+          ),
```
Comment (ArturDolzan) [inline webapp/app/[locale]/tunnel/_components/reviewOperation/step.tsx:73]: I think we don't need this fragment

```suggestion
          children: (
             <span className="mr-auto text-neutral-600">{description}</span>
          ),
```

**Review by ArturDolzan — COMMENTED** [review]
A minor comment otherwise LGTM.

File: webapp/app/[locale]/tunnel/_components/reviewOperation/step.tsx (hunk 1 above)
Comment (gndelia) [reply webapp/app/[locale]/tunnel/_components/reviewOperation/step.tsx:73]: You're correct, it is not needed indeed!

Comment (gndelia) [reply webapp/app/[locale]/tunnel/_components/reviewOperation/step.tsx]: Removed in 35cd5f2

**Pull request merged**